app/src/main/kotlin/com/simplemobiletools/notes/pro/activities/MainActivity.kt

```diff
@@ -114,6 +267,7 @@ class MainActivity : SimpleActivity() {
             findItem(R.id.open_note).isVisible = shouldBeVisible
             findItem(R.id.delete_note).isVisible = shouldBeVisible
             findItem(R.id.export_all_notes).isVisible = shouldBeVisible
+            findItem(R.id.open_search).isVisible = view_pager.currentItem != 0
```

Comment:
why isnt search shown at the default page?

Reply:
fixed